### PR TITLE
Merging to release-4-lts: when adding a new certificate check if exists by using the Exist method and not Get (#4003)

### PR DESCRIPTION
**Pull request by buger**
when adding a new certificate check if exists by using the Exist method and not Get (#4003)

* when adding a new certificate check if exists by using the Exist method and not Get

* gofmt & goimports

Co-authored-by: Jeff <jeffy.mathew100@gmail.com>

### CODE REVIEW

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/4787/merge
Commit:  
Triggered by: pull_request (@buger)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/4223928046)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/4787/merge
Commit: 94f0991db33786db82911c3ad4c62a7090cac2a5 
Triggered by: pull_request (@sredxny)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/4224122907)

**Comment by Tyk-ITS** [comment]
**API tests result: failure** :no_entry_sign: 
Branch used: refs/pull/4787/merge
Commit: ddc8cd85f8fade4847b1ba9df1e7fb0a74935d21 
Triggered by: pull_request (@sredxny)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/4224207521)

File: certs/manager.go [hunk 1]
```diff
@@ -498,7 +498,8 @@ func (c *CertificateManager) Add(certData []byte, orgID string) (string, error)
 	}
 	certID = orgID + certID
 
-	if cert, err := c.storage.GetKey("raw-" + certID); err == nil && cert != "" {
+	// use `Exist` rather than `Get` to avoiding infinite looping in mdcb storage
```
Comment (mativm02) [inline certs/manager.go:501]: The comment should be: "use `Exist` rather than `Get` to **avoid** infinite looping in MDCB storage"

**Comment by Tyk-ITS** [comment]
**API tests result: skipped** :no_entry_sign: 
Branch used: refs/heads/release-4-lts
Commit: ecab676e82fb942cb4c54ec16a44853f1e8ae112 Merging to release-4-lts: when adding a new certificate check if exists by using the Exist method and not Get (#4003) (#4787)

when adding a new certificate check if exists by using the Exist method
and not Get (#4003)

* when adding a new certificate check if exists by using the Exist
method and not Get

* gofmt & goimports

Co-authored-by: Jeff <jeffy.mathew100@gmail.com>

---------

Co-authored-by: Sredny M <sredny.buitrago@gmail.com>
Triggered by: push (@sredxny)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/4225567174)

**Pull request merged**